.github/ISSUE_TEMPLATE/add-instance.yaml

```diff
@@ -48,7 +48,7 @@ body:
       label: Source code URL
       description: |
         If you have modified the source code, please publish it to an URL and make sure that the URL is publicly accessible in your instance's site
-        ( Check your [settings.yml](https://github.com/searx/searx/blob/e88bea53ebf220648685cdb8eff017357728494e/searx/settings.yml#L8) )
+        ( Check your [buildenv](https://docs.searxng.org/dev/makefile.html#make-buildenv) )
```

Comment:
It won't work for docker installation.

Reply:
:-) .. this was the point where I opened https://github.com/searxng/searxng/issues/728

Please push your suggestion on top of this branch / Sorry, I do not have more time today.

Reply:
I don't understand how it is related and how it helps for the docker installation.

There is no usable `manage` script with the docker image, only `settings.yml`.

Reply:
See c102320 : the comment asks to check git_url for searx instance (SearXNG relies on the git command to get the GIT URL, there is nothing to do). We can update this field later.

Reply:
> there is nothing to do

Thanks, c102320 was exactly what I asked for.

> I'm going to merge this PR 

Perfect .. this was what I expected :-)